PowershellTools.ProcessManager.Data/Common/Constants.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PowershellTools.ProcessManager.Data.Common
+{
+    internal static class Constants
+    {
+        public const string ProcessManagerHostUri = "net.pipe://localhost/";
+
+        public const string ProcessManagerHostRelativeUri = "NamedPipePowershellProcess";
+
+        public const string ProcessManagerDataNamespace = "https://github.com/Microsoft/poshtools";
+
+        public const string ReadyEventPrefix = "VspowershelltoolProcess:";
+
+        // 1K in milliseconds
+        public const int HostProcessStartupTimeout = 100000;
+
+        public const string PowershellHostExeName = "PowershellToolsProcessHost.exe";
+
+        // 10M in bytes
+        public const int BindingMaxReceivedMessageSize = 10000000;

```

Comment:
The timeout should be 1000. I make an estimate about the waiting time for host process to be initialized and thought 1 second should be tolerable. The message size is all about script size. 10M should be enough. We can talk about this later as it's not a final call:)